### PR TITLE
Update bundle_de.properties to latest bundle

### PR DESCRIPTION
**Pull request by hype11**
add new keys and translate them

### CODE REVIEW

**Comment by LonwoLonwo** [comment]
Hello @hype11 

Please, next time, make your changes in one PR. The logic is the same for all of them.

File: plugins/org.jkiss.dbeaver.data.transfer.ui/OSGI-INF/l10n/bundle_de.properties [hunk 1]
```diff
@@ -1,4 +1,9 @@
+extension-point.org.jkiss.dbeaver.dataTransferConfigurator.name = Daten\u00FCbertragung Konfigurationen
+
 command.org.jkiss.dbeaver.core.export.data.description = Daten in Dateien oder Datenbanktabellen exportieren
 command.org.jkiss.dbeaver.core.export.data.name        = Daten exportieren
 command.org.jkiss.dbeaver.core.import.data.description = Daten aus Datenbanktabellen importieren
 command.org.jkiss.dbeaver.core.import.data.name        = Daten importieren
+
+pref.page.name.datatransfer=Daten\u00FCbertragung
+keyword.org.jkiss.dbeaver.preferences.datatransfer=Daten\u00FCbertragung \u00DCbertragung Fallback
```
Comment (LonwoLonwo) [inline plugins/org.jkiss.dbeaver.data.transfer.ui/OSGI-INF/l10n/bundle_de.properties:9]: It is better to have English keywords here as well as German keywords.
And does the "Fallback" word untranslatable?

File: plugins/org.jkiss.dbeaver.data.transfer.ui/OSGI-INF/l10n/bundle_de.properties [hunk 2]
```diff
@@ -1,4 +1,9 @@
+extension-point.org.jkiss.dbeaver.dataTransferConfigurator.name = Daten\u00FCbertragung Konfigurationen
```
Comment (LonwoLonwo) [inline plugins/org.jkiss.dbeaver.data.transfer.ui/OSGI-INF/l10n/bundle_de.properties]: Why is it not "Konfiguratoren"?

File: plugins/org.jkiss.dbeaver.data.transfer.ui/OSGI-INF/l10n/bundle_de.properties (hunk 1 above)
Comment (hype11) [reply plugins/org.jkiss.dbeaver.data.transfer.ui/OSGI-INF/l10n/bundle_de.properties:9]: Sometimes a translation brings more confusion than clarification. Here in technical software, fallback is well understood, even in Germany

File: plugins/org.jkiss.dbeaver.data.transfer.ui/OSGI-INF/l10n/bundle_de.properties (hunk 2 above)
Comment (hype11) [reply plugins/org.jkiss.dbeaver.data.transfer.ui/OSGI-INF/l10n/bundle_de.properties]: Where the string is showed in dbeaver? "Konfiguratoren" is the configurator ui and "Konfigurationen" is the configuration data itself at whole

Comment (LonwoLonwo) [reply plugins/org.jkiss.dbeaver.data.transfer.ui/OSGI-INF/l10n/bundle_de.properties]: No, I can't find it in UI. But it still - data Transfer Configurator not configuration.

Comment (hype11) [reply plugins/org.jkiss.dbeaver.data.transfer.ui/OSGI-INF/l10n/bundle_de.properties]: I changed it to "Konfigurator". But why is there a string that is not displayed in the user interface?

Comment (LonwoLonwo) [reply plugins/org.jkiss.dbeaver.data.transfer.ui/OSGI-INF/l10n/bundle_de.properties]: For historical reasons.
And again: Why is it not "Konfiguratoren"? Why is it not a plural form?

Original string is:
`extension-point.org.jkiss.dbeaver.dataTransferConfigurator.name = Data Transfer UI Configurators`

Comment (hype11) [reply plugins/org.jkiss.dbeaver.data.transfer.ui/OSGI-INF/l10n/bundle_de.properties]: I changed it to Konfiguratoren

**Review by LonwoLonwo — APPROVED** [review]

**Review by E1izabeth — APPROVED** [review]

**Pull request merged**